pkg/spdk/disk.go

```diff
@@ -159,13 +159,13 @@ func getDiskPath(path string) string {
 }
 
 func getDeviceNumber(filename string) (string, error) {
-	fi, err := os.Stat(filename)
+	fileInfo, err := os.Stat(filename)
 	if err != nil {
 		return "", err
 	}
-	dev := fi.Sys().(*syscall.Stat_t).Dev
-	major := int(dev >> 8)
-	minor := int(dev & 0xff)
+	dev := fileInfo.Sys().(*syscall.Stat_t).Rdev
```

Comment:
Can you explain the purpose of this change?

Reply:
`sdev` is the ID of device containing file.
`rdev` is the device ID.

The function is getting the device ID (major-minor) of the give disk path.